data/brands/amenity/fast_food.json

```diff
@@ -8760,6 +8760,7 @@
         "brand": "八方雲集",
         "brand:en": "Eight Way",
         "brand:wikidata": "Q28417381",
+        "brand:wikipedia": "zh:八方雲集",
```

Comment:
@Supaplextw In yesterday‘s https://github.com/osmlab/name-suggestion-index/pull/7079#discussion_r979427614, you don't need to add `brand:wikipedia` in NSI